soda_machine.rb

```diff
@@ -7,12 +7,24 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.count
   end
 
   def find_soda(soda_brand)
```

Comment:
The intention is to return an instance of the Soda class. So what you did is correct. Nice work here.